meta-avocado-raspberrypi/conf/machine/avocado-raspberrypi.inc

```diff
@@ -1,4 +1,5 @@
-WKS_FILE ?= "avocado-raspberrypi.wks"
+# WKS_FILE ?= "avocado-raspberrypi.wks"
+UBOOT_ENV_SIZE = "0x20000"
```

Comment:
Where are you pulling U-Boot from now that `meta-raspberrypi/recipes-bsp/u-boot/` is masked out?

See `RPI_USE_U_BOOT = "1"` below.